src/PKSim.CLI.Core/Services/QualificationRunner.cs

```diff
@@ -103,6 +103,11 @@ public async Task RunBatchAsync(QualificationRunOptions runOptions)
          var projectFile = Path.Combine(projectOutputFolder, $"{project.Name}{CoreConstants.Filter.PROJECT_EXTENSION}");
          _workspace.Project = project;
          _workspacePersistor.SaveSession(_workspace, projectFile);
+         _logger.AddDebug($"Project saved to '{projectFile}'", project.Name);
+
+         var snapshotFile = Path.Combine(projectOutputFolder, $"{project.Name}{Constants.Filter.JSON_EXTENSION}");
```

Comment:
I am exporting the snapshot from the project again so that we have the latest and greatest based on the project